tools/FaultInjectionService/AUTHORS

```diff
@@ -0,0 +1,12 @@
+Current Maintainer                                                              
```

Comment:
I am not sure if we need this file (but I can be convinced). Until now we didn't follow this pattern. There are no maintainers of SCM / Freon / OM. The community itself is the maintainers but we can check the git history for first contact.

This file is also requires additional maintenance. I would suggest to remove it.

Reply:
@elek Thank you for reviewing it. 
- Yes I will remove AUTHORS file and change the directory name to lower case.
- Its supported only on linux and some dependencies require building them from the source. I will add an automated build script as next set of enhancement and create a separate PR.